react-web/src/components/CreateCertificate/CreateCertificate.tsx

```diff
@@ -0,0 +1,157 @@
+
+import Button from "components/Button/Button";
+import Modal from "components/Modal/Modal";
+
+
+import { Address,
+    Value,
+    BigNum,
+    LinearFee,
+    TransactionBuilderConfigBuilder,
+    TransactionUnspentOutputs,
+    TransactionUnspentOutput,
+    TransactionBuilder,
+    TransactionWitnessSet,
+    Transaction,
+    TransactionOutput,
+    CoinSelectionStrategyCIP2
+ } from '@emurgo/cardano-serialization-lib-asmjs';
+import Toast from "components/Toast/Toast";
+import { fetchData } from "api/api";
+
+let Buffer = require("buffer/").Buffer;
+
+const CreateCertificate = () => {
+    const { uuid } = useAppSelector((state) => state.certification);
+    const { address, wallet } = useAppSelector((state) => state.auth);
+    const [ certifying, setCertifying ] = useState(false);
+    const [ certified, setCertified ] = useState(false);
+    const [ transactionId, setTransactionId ] = useState("")
+    const [ showError, setShowError ] = useState("");
+    const [ openModal, setOpenModal ] = useState(false);
+    const [ disableCertify, setDisableCertify ] = useState(false);
+
+    const onCloseModal = () => { setOpenModal(false) }
+
+    const handleError = (errorObj: any) => {
+        let errorMsg = ''
+        if (typeof errorObj === 'string') {
+            errorMsg = errorObj + ' Please try again.'
+        } else if (errorObj?.info) {
+            errorMsg = errorObj.info + ' Please try again.'
+        } else if (errorObj?.response?.message) {
+            errorMsg = errorObj?.response.message + ' Please try again.'
+        } else if (errorObj?.response?.data) {
+            errorMsg = errorObj.response.statusText + ' - ' + errorObj.response.data 
+        }
+        setShowError(errorMsg.length > 50 ? 'Something wrong occurred. Please try again later.' : errorMsg);
+        setTimeout(() => { setShowError("") }, 5000)
+        setCertifying(false);
+        if (errorObj?.response?.status === 403) {
+            setDisableCertify(true)
+        }
+    }
+
+    const triggerSubmitCertificate = async (txnId: string) => {
+        const response: any = await fetchData.post('/run/' + uuid + '/certificate' + '?transactionid=' + txnId).catch(handleError)
+        try {
+            console.log('broadcasted tnx data ', response.data);
+            setTransactionId(response.data.transactionId)
+            setOpenModal(true)
+            setCertifying(false)
+            setCertified(true)
+        } catch(e) { }
+    }
+
+    const triggerGetCertificate = async () => {
+        setCertifying(true);
+        setShowError("")
+        try {
+            // const walletAddressRes: any = await fetchData.get('/wallet-address').catch(handleError)
+            // const applicationWallet_receiveAddr = walletAddressRes.data;
+            /** For mock */
+            const applicationWallet_receiveAddr = 'addr_test1qz2rzeqq8n82gajfp35enq3mxhaynx6zhuql2c7yaljr25mfaznfszxu8275k6v7n05w5azzmxahfzdq564xuuyg73pqnqtrrc'
```

Comment:
there is already an endpoint on the BE for this. why not using it?

Reply:
Uncommented above lines, and commented this.